spec/rubocop/cop/style/accessor_grouping_spec.rb

```diff
@@ -264,6 +264,16 @@ class Foo
         end
       RUBY
     end
+
+    it 'does not register an offense for grouped accessors having rbs-inline annotation' do
+      expect_no_offenses(<<~RUBY)
+        class Foo
+          attr_reader :one #: String
+
+          attr_reader :two, :three
+        end
+      RUBY
+    end
```

Comment:
Can you add the test?

```suggestion
    end

    it 'registers an offense for grouped accessors having non-RBS::Inline annotation' do
      expect_offense(<<~RUBY)
        class Foo
          attr_reader :one # comment #: String
          ^^^^^^^^^^^^^^^^ Group together all `attr_reader` attributes.

          attr_reader :two, :three
          ^^^^^^^^^^^^^^^^^^^^^^^^ Group together all `attr_reader` attributes.
        end
      RUBY

      expect_correction(<<~RUBY)
        class Foo
          attr_reader :one, :two, :three # comment #: String
        end
      RUBY
    end    
```